opencxl/apps/accelerator.py

```diff
@@ -293,19 +304,19 @@ async def _run(self):
         tasks = [
             create_task(self._sw_conn_client.run()),
             create_task(self._cxl_type1_device.run()),
-            create_task(self._irq_manager.run()),
+            # create_task(self._irq_manager.run()),
```

Comment:
Why does this have to be commented out?

Reply:
Because _irq_manager is undefined due to the early return of __init__. It should be enabled in the future.